packages/util/src/address/sshash.ts

```diff
@@ -1,10 +1,13 @@
-// Copyright 2017-2019 @polkadot/keyring authors & contributors
+// Copyright 2017-2019 @polkadot/util authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { stringToU8a, u8aConcat } from '@polkadot/util';
+// This is not great - here we now have a dependency from util -> crypto
 import { blake2AsU8a } from '@polkadot/util-crypto';
```

Comment:
This is the part that causes all hell to break loose (Comment actually made before seeing the failures)

Only solution would be to move the address/* helpers to util-crypto